pkg/pretty-c/.BUILDINFO

```diff
@@ -0,0 +1,1856 @@
+format = 2
+pkgname = pretty-c
+pkgbase = pretty-c
+pkgver = 1.0.0-1
+pkgarch = any
+pkgbuild_sha256sum = 7e535f1ee48a678dccd9b579280a2f152dc7b45d5fdecfddb9149c038569ed29
+packager = Unknown Packager
+builddate = 1727266348
+builddir = /home/wafi/coding/pretty.c
+startdir = /home/wafi/coding/pretty.c
+buildtool = makepkg
+buildtoolver = 7.0.0
+buildenv = !distcc
+buildenv = color
+buildenv = !ccache
+buildenv = check
+buildenv = !sign
+options = strip
+options = docs
+options = !libtool
+options = !staticlibs
+options = emptydirs
+options = zipman
+options = purge
+options = debug
+options = lto
+installed = a52dec-0.8.0-2-x86_64
```

Comment:
I'm not sure all of these are needed to "build" pretty.c. Can you cull the list down?

Reply:
no, it's just like that no matter what

Reply:
Okay then.